assets/js/googlesitekit/data/create-fetch-store.js

```diff
@@ -163,8 +170,13 @@ export const createFetchStore = ( {
 			} catch ( e ) {
 				error = e;
 
+				const { receiveError } = registry ? registry.dispatch( storeName ) : {};
+				if ( receiveError ) {
+					yield receiveError( error, baseName, args.length ? args : undefined );
```

Comment:
@eugene-manuilov Related to https://github.com/google/site-kit-wp/issues/1814#issuecomment-670791342:
```suggestion
					yield receiveError( error, baseName, args.length ? args : undefined );
					// @TODO: Remove the following once all instances of the legacy behavior have been removed.
					yield receiveError( error );
```

Reply:
:+1: Updated.